test/integration/enumerating-databases/list_databases.test.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
+
+import { AddUserOptions, MongoClient, MongoServerError } from '../../../src';
+
+describe.only('listDatabases', function () {
+  // TODO(NODE3860): Create driver test variants that run with AUTH enabled
+  // beforeEach(function () {
+  //   if (process.env.AUTH !== 'auth') {
+  //     this.skip();
+  //   }
+  // });
```

Comment:
```suggestion
  beforeEach(function () {
    if (process.env.AUTH !== 'auth') {
      this.skip();
    }
  });
```

We still need the skip, but the comment LGTM

Reply:
sorry uncommented to test, putting back.